knowledge-portal/central/README.md

```diff
@@ -1,48 +1,60 @@
-# Setting up pipeline in central repository
+# Knowledge Portal Central Pipeline
 
->You can just simply use the below code and paste it in your `.gitlab-ci.yml` file of your repo.
+## Usage
+
+### Adding this pipeline to your repo
+
+You will have to modify the existing `gitlab-ci.yml` file to include:
 
 ```yaml
 stages:
-    - push
+  - push
+
 include:
-    - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/central/.gitlab-ci.yml'
+  - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/central/.gitlab-ci.yml'
+
+variables: 
+  GIT_STRATEGY: clone
+
 ```
 
-## Setting up Access Token in central repository
+### Central Pipeline Checks
 
-1. Go to `Settings >> Access Tokens`.
-2. Set `Token name = PUSH_TOKEN`. This is important and you can't choose any other name without modifying the source pipeline as well.
-3. Select the role `Maintainer` and `write_repository` as the scope of token.
-4. `Create project access token` will create a unique token for you that can then be shared with `maintainers` setting up individual `source` pipelines.
+Before the job in pipeline pushes files back to `source`, it checks multiple things based on your mapping file. Knowing about these checks will help you use the pipeline easily and make any necessary changes before any change on wikijs become available at `source` repository.
 
+- The jobs will only run when a commit happens on `main` branch with a message starting with `docs:`, This ensures that the jobs will only run when a commit has been made from wikijs portal.
+- Ensure that all image URL paths in Markdown files are absolute and begin with `/`.
```

Comment:
I presume this is for local image files, not ones on the web (Ex: `http` or `https`). If yes, can we specify that as a subpoint?

Reply:
done